Source/Plugins/SyntheticBuilding/SyntheticBuildingFilters/InsertPrecipitatePhases.h

```diff
@@ -323,7 +323,7 @@ class SyntheticBuilding_EXPORT InsertPrecipitatePhases : public AbstractFilter
    * @brief insert_precipitate Performs the insertion of a precipitate into the packing volume
    * @param featureNum Id for the precipitate to be inserted
    */
-  void insert_precipitate(size_t featureNum);
+  void insert_precipitate(size_t gnum);
```

Comment:
What is the reason for this change?  featureNum tells me a lot more about the meaning and seems to be correct based on the description.  Does g have a specific meaning for precipitates that I should know about?